src/modules/reward/mocks/index.js

```diff
@@ -0,0 +1,7 @@
+import { rest } from 'msw';
+import { API_VERSION } from 'src/const/config';
+import { mockRewardConstants, mockRewardInflation } from 'src/modules/reward/__fixtures__';
+
+export const rewardConstants = rest.get(`*/api/${API_VERSION}/reward/constants`, async (req, res, ctx) => res(ctx.delay(20), ctx.json(mockRewardConstants)));
+
```

Comment:
space can be removed

Reply:
Why? Isnt it nicer with a space in the end of files.Better diffs in the future when adding to this file

Reply:
ah, or between the functions you mean? I think it is more readable to have a space between, other mock endpoints have a space as well